composer.json

```diff
@@ -21,13 +21,13 @@
     "license": "MIT",
     "name": "php-fp/php-fp-either",
     "scripts": {
-        "test": "vendor/bin/phpunit test"
+        "test": "phpunit test"
```

Comment:
Composer bin directory is [automagically resolved for scripts](https://getcomposer.org/doc/articles/scripts.md#writing-custom-commands).